keeper/state_portal.go

```diff
@@ -23,11 +23,18 @@ package keeper
 import (
 	"context"
 
+	"cosmossdk.io/collections"
 	"cosmossdk.io/errors"
 
 	"dollar.noble.xyz/types/portal"
 )
 
+// GetPortalPaused is a utility that returns the current paused state.
+func (k *Keeper) GetPortalPaused(ctx context.Context) bool {
+	paused, _ := k.PortalPaused.Get(ctx)
```

Comment:
Do we want to ignore the error here?

Reply:
We ignore this because then the value doesn't need to be set in genesis (like we have for state in the FiatTokenFactory). This makes running a chain a lot easier, without having to worry about customizations to the genesis file.

This is safe because if an error is encountered, it would be if the key isn't found (meaning we're not paused) or if the value fails unmarshalling (which is impossible as the only way the value is set is via a bool in the `MsgSetPausedState` method)!